src/test/java/org/broadinstitute/hellbender/tools/walkers/vqsr/VariantRecalibratorIntegrationTest.java

```diff
@@ -44,6 +45,64 @@
+                    getLargeVQSRTestDataDir() + "Omni25_sites_1525_samples.b37.20.1M-10M.vcf",
+                    "-an", "QD", "-an", "HaplotypeScore", "-an", "HRun", "-an", "HRun",
+                    "--trust-all-polymorphic", // for speed
+                    "-mode", "SNP",
+                    "--" + StandardArgumentDefinitions.ADD_OUTPUT_VCF_COMMANDLINE, "false"
+            };
+
+    private final String[] VQSRBothParamsWithResources =
+            new String[] {
+                    "--variant",
+                    getLargeVQSRTestDataDir() + "phase1.projectConsensus.chr20.1M-10M.raw.snps.vcf",
+                    "--variant",
+                    getLargeVQSRTestDataDir() + "g94982_20_1m_10m_python_2dcnn.indels.vcf.gz",
+                    "-L","20:1,000,000-10,000,000",
+                    "--resource:known,known=true,prior=10.0",
+                    getLargeVQSRTestDataDir() + "dbsnp_132_b37.leftAligned.20.1M-10M.vcf",
+                    "--resource:truth_training1,truth=true,training=true,prior=15.0",
+                    getLargeVQSRTestDataDir() + "sites_r27_nr.b37_fwd.20.1M-10M.vcf",
+                    "--resource:truth_training2,training=true,truth=true,prior=12.0",
+                    getLargeVQSRTestDataDir() + "Omni25_sites_1525_samples.b37.20.1M-10M.vcf",
+                    "-an", "QD", "-an", "HaplotypeScore", "-an", "HRun",
+                    "--trust-all-polymorphic", // for speed
+                    "-mode", "BOTH",
+                    "--" + StandardArgumentDefinitions.ADD_OUTPUT_VCF_COMMANDLINE, "false",
+                    "--max-gaussians", "6"
+            };
+
+    private final String[] VQSRBothAggregateParamsWithResources =
+            new String[] {
+                    "--variant",
+                    getLargeVQSRTestDataDir() + "phase1.projectConsensus.chr20.1M-10M.raw.snps.vcf",
+                    "--aggregate",
+                    getLargeVQSRTestDataDir() + "g94982_20_1m_10m_python_2dcnn.indels.vcf.gz",
+                    "-L","20:1,000,000-10,000,000",
+                    "--resource:known,known=true,prior=10.0",
+                    getLargeVQSRTestDataDir() + "dbsnp_132_b37.leftAligned.20.1M-10M.vcf",
+                    "--resource:truth_training1,truth=true,training=true,prior=15.0",
+                    getLargeVQSRTestDataDir() + "sites_r27_nr.b37_fwd.20.1M-10M.vcf",
+                    "--resource:truth_training2,training=true,truth=true,prior=12.0",
+                    getLargeVQSRTestDataDir() + "Omni25_sites_1525_samples.b37.20.1M-10M.vcf",
+                    "-an", "QD", "-an", "HaplotypeScore", "-an", "HRun",
+                    "--trust-all-polymorphic", // for speed
+                    "-mode", "BOTH",
+                    "--" + StandardArgumentDefinitions.ADD_OUTPUT_VCF_COMMANDLINE, "false",
+                    " --output %s" + " -tranches-file %s",
+                    "--max-gaussians", "6"
+            };
+
```

Comment:
Minor nit - it would be nice if the common part of the command line string was factored out of these, since it takes some staring to spot the subtle differences (ie, the duplicate annotation). Up to you. No good deed goes unpunished...